src/combinator.ts

```diff
@@ -1,5 +1,6 @@
 import { pureP, TypeProxy } from '.';
 import { ParseError } from './error';
+import { undefinedP } from './primitive';
```

Comment:
You should increment the version number in package.json

Reply:
done

Reply:
nice